azure-pipelines.yml

```
@@ -23,61 +21,14 @@ resources:
       ref: refs/heads/master
       endpoint: 'GitHub (SFA)'
 
-steps:
-  - task: gittools.gittools.setup-gitversion-task.gitversion/setup@0
-    displayName: gitversion/setup
-    inputs:
-      versionSpec: 5.x
+stages:
+- stage: Build
+  jobs:
+  - template: pipeline-templates\job\code-build.yml
```

```suggestion
  - template: pipeline-templates/job/code-build.yml
```
Forward slashes so can run on linux as well